multisig/proposals/opsepolia/vip-001/index.ts

```diff
@@ -0,0 +1,184 @@
+      params: [],
+    },
+    {
+      target: opsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[WBTC, CHAINLINK_BTC_FEED, STALE_PERIOD_30M]],
+    },
+    {
+      target: opsepolia.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          WBTC,
+          [
+            opsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: opsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[WETH, CHAINLINK_ETH_FEED, STALE_PERIOD_30M]],
+    },
+    {
+      target: opsepolia.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          WETH,
+          [
+            opsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: opsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[USDC, CHAINLINK_USDC_FEED, STALE_PERIOD_30M]],
+    },
+    {
+      target: opsepolia.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          USDC,
+          [
+            opsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: opsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[USDT, CHAINLINK_USDT_FEED, STALE_PERIOD_30M]],
+    },
+    {
+      target: opsepolia.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          USDT,
+          [
+            opsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: opsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[OP, CHAINLINK_OP_FEED, STALE_PERIOD_26H]],
+    },
+    {
+      target: opsepolia.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          OP,
+          [
+            opsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
```

Comment:
I would also configure the price feed for XVS, fixed to $7:

```
    {
      target: opsepolia.CHAINLINK_ORACLE,
      signature: "setDirectPrice(address,uint256)",
      params: [opsepolia.XVS, "7000000000000000000"], // 7$
    },

    {
      target: opsepolia.RESILIENT_ORACLE,
      signature: "setTokenConfig((address,address[3],bool[3]))",
      params: [
        [
          opsepolia.XVS,
          [opsepolia.CHAINLINK_ORACLE, "0x0000000000000000000000000000000000000000", "0x0000000000000000000000000000000000000000"],
          [true, false, false],
        ],
      ],
    },
```

Reply:
And I would add a simulation check for it

Reply:
We need to use Redstone here for XVS, as mentioned in notion doc.

Reply:
RedStoneOracle and ChainlinkOracle share the codebase, so we could use any of them. I suggested using the ChainlinkOracle to avoid an extra deployment of a new contract